docs/metrics.md

```diff
@@ -0,0 +1,171 @@
+# METRICS
+
+## Data Analysis
+The collected data will primarily be used to answer the following questions.
+Images are used for visualization and are not composed of actual data.
+
+### Do users install and run this?
+
+What is the overall engagement of the Containers experiment?
+**This is the standard Daily Active User (DAU) and Monthly Active User (MAU) analysis.**
+
+This captures data from the users who have the add-on installed, regardless of
+whether they are actively interacting with it.
+
+![](images/kpi-1.png)
+
+### Immediate Questions
```

Comment:
It may be good to review some metrics that should not change. For example, search counts, active hours, and page loads should show no change or an increase.

Will this experiment affect the way standard telemetry data is collected? E.g. this will not create a new client_id for each container, right?